.tutone.yml

```diff
@@ -183,49 +184,147 @@ packages:
       - name: EdgeTraceObserverStatus
       - name: EdgeTracing
 
-  - name: infrastructure
-    path: pkg/infrastructure
-    import_path: github.com/newrelic/newrelic-client-go/pkg/infrastructure
-    generators:
-      - typegen
-    types: []
 
   - name: entities
     path: pkg/entities
     import_path: github.com/newrelic/newrelic-client-go/pkg/entities
     generators:
       - typegen
+      - nerdgraphclient
+    imports:
+      - github.com/newrelic/newrelic-client-go/pkg/accounts
+      - github.com/newrelic/newrelic-client-go/pkg/nrdb
+      - github.com/newrelic/newrelic-client-go/pkg/nrtime
+    queries:
+      - path: ["actor"]
+        endpoints:
+          - name: entities
+            max_query_field_depth: 2
+          - name: entity
+            max_query_field_depth: 2
+          - name: entitySearch
+            max_query_field_depth: 2
+            include_arguments:
+              - "queryBuilder"
+    mutations:
+      - name: taggingAddTagsToEntity
+        max_query_field_depth: 1
+      - name: taggingDeleteTagFromEntity
+        max_query_field_depth: 1
+      - name: taggingDeleteTagValuesFromEntity
+        max_query_field_depth: 1
+      - name: taggingReplaceTagsOnEntity
+        max_query_field_depth: 1
+
     types:
       - name: AttributeMap
         create_as: "map[string]interface{}"
-#      - name: Entity
-#      - name: EntityOutline
+
+      # Types this package owns
+      - name: AlertableEntity
```

Comment:
Since you have the `queries` above set for this package, the expander should drag in all these types without having to specify them manually here.  Are they here just for posterity?

Reply:
Basically?  One item I've been thinking a lot about is a way for Tutone to understand cross-package deps.  The initial thought was if you statically define the type, then that package "Owns" it, and therefore if required by another package, tutone could import/auto-reference the other package and the type.  None of that has been done in Tutone, so having them here is not explicitly required, but somewhat serves a manual purpose of claiming them...